### ISSUE TITLE
Quote marks in file names

### ISSUE THREAD

**Issue by rbeezer** [openwebwork/webwork2#652]
Publisher's problem sets from Rogawski Calculus have directory names like "Green's_Theorem".

Problems work fine, but File Mnager complains about an illegal filename, and I cannot descend into that directory to move an image file to a local (edited) copy.  Looks to me like it is the quote mark.  Not sure if it is a Unicode character, but it looks like plain ASCII to me.  I'm on hosted2 at Rochester.

**Comment by jwj61** [comment]
Are these OPL problems, or from the publisher.

For the OPL, I thought we made copies of problems with bad paths and marked the bad ones as "don't show in the library browser".  For those, don't use them -- use the ones with matching paths with bad characters removed.

**Comment by rbeezer** [comment]
John,

These are the ones from the publlisher, though I think they put a small subset into the OPL.  So I guess this is beyond WW's control.  I don't know how the File Manager works on the back-end, but I thought maybe wrapping a filename with the right kind of quotes would help protect against a lot of this sort of thing.

Rob

**Comment by rbeezer** [comment]
Is this not the webwork2 repository?  The banner says "Course management front end".

**Comment by dpvc** [comment]
@rbeezer, you are right, the quote mark is a problem, and the file manager won't let you use that in a file or directory name.  There is no quoting that will allow it.

One possible solution would be to make an archive of the directory containing the Green's_Theorem directory, and then download that to your computer, where you can extract the file locally and upload it again.  A bit of a pain, but it should work, if you have write access to the directory where the archive will be made.  Worth a shot.

**Comment by rbeezer** [comment]
Thanks for the workaround, @dpvc.    Sorry to hear no amount of quoting will help.

The publisher's problems have some absolute tolerances that are impractical, so I have to edit the problems to make them relative.  But then I have to manually move the corresponding image files "by hand."  Which Mike Gage says will be automatic in the future.

Since I am almost done with setup for the semester, I'm just going to pick a different problem.  But I'll add the quoate mark problem to my list for the publisher.  ;-)

Rob
